spec/index.html

```diff
@@ -319,50 +319,87 @@ <h3>RDF Vocabularies and Namespace IRIs</h3>
+      
+<!-- When read literally, one could understand this to say that a triple term _references_ (whatever references may mean here) a proposition. But, I think it is simply meant that a triple term encodes a proposition, just like an RDF triple (?) If so, then perhaps the ambiguity could be removed by the following. 
+-->
+<!-- original:
+      This use is a reference to a <a>proposition</a>. 
+-->
+As an [=RDF triple=], it encodes a <a>proposition</a>.
+    
+<!-- To reduce the complexity of the introduction, I propose moving the parts on asserted vs. non-asserted (including the part on triple annotations) and referential transparency to later...-->
 
     <p>
       A <dfn>reifying triple</dfn> is a triple where the <a>predicate</a> is <code>rdf:reifies</code>
       and the <a>object</a> is a <a>triple term</a>.
       The <a>subject</a> of that triple is called a <dfn>reifier</dfn>, and it can be the
       subject or object of other triples.
-      When the triple term of a <a>reifying triple</a> also [=appears=] in the
-      same graph as an <a>asserted triple</a>, the subset of triples that share
-      the same <a>reifier</a> as subject is called a <dfn>triple annotation</dfn>.
+<!-- 
+    The descriptions here are very abstract.
+    This part describes how it could be useful to use reifiers in other triples.
+    I realize that this may be considered "non-normative" (and that there is a note on it later, and that the figures already illustrate this). 
+    But I feel that the text should also have an indication as to why one would want do such a thing.
+-->
+     For instance, those other triples, i.e., which also include the [=reifier=] as subject or object, can provide metadata about the [=reifier=]'s associated [=triple term=].
     </p>
 
     <p>
-      Concrete syntaxes, such as Turtle [[RDF12-TURTLE]], may have shortcuts
-      for specifying <a>reifying triples</a> and <a>triple annotations</a> more
-      succinctly.
-    </p>
-
-    <p>The following diagram represents a <a>reifying triple</a> of an unasserted, abstract <a>triple term</a>, and a <a>triple</a> relating the <a>reifier</a> to another resource.</p>
+<!-- Removing the "unasserted, abstract" part as this is now discussed below. (Also, note that "abstract" had not been used yet to describe triple terms; so, a reader could wonder whether "abstract" triple terms are the same as triple terms, or different). Also, as above, I slightly rewrote part of it to describe why the second triple could be useful.
+-->
+<!-- original:  
+The following diagram represents a <a>reifying triple</a> of an unasserted, abstract <a>triple term</a>, and a <a>triple</a> relating the <a>reifier</a> to another resource. 
+-->
+      For instance, the following diagram represents a <a>reifying triple</a> of a <a>triple term</a>, together with a <a>triple</a> including the <a>reifier</a> as subject, which provides metadata about the <a>triple term</a> (here, describing provenance).</p>
```

Comment:
Same problematic use of "metadata about the triple term".

(Also (perhaps due to all type/token discussions), "for instance" struck me as odd there.  I'd prefer "for example". Looking through the document, "for instance" is used in two places, and "for example" in 18. Not that I'm against variation, so my main point is the situational risk of misimplication in association with reification.)

Reply:
fair enough and I replaced the "for instance" in this section with "for example".